app/models/comment.rb

```diff
@@ -5,7 +5,7 @@ class Comment < ApplicationRecord
   after_create :update_post_comments_counter
   after_destroy :update_post_comments_counter
 
-  private
+  # private
```

Comment:
- [ ] Kindly remove all the commented code to make the code more readable and clean.